tslint.json

```diff
@@ -111,13 +111,6 @@
           }
         ],
         "Directive": [
-          {
-            "argument": 0,
-            "properties": {
-              "!host": "\\[class\\]"
```

Comment:
Context for this lint check is that `[class]` bindings in ViewEngine would clobber static classes. This isn't the case with Ivy and we actually had a binding like this for a while in the `MAT_BUTTON_HOST` constant, the lint rule just wasn't picking it up.